.github/workflows/continuous-delivery.yml

```diff
@@ -2,160 +2,33 @@ name: Continuous Delivery
 
 on:
   workflow_dispatch:
-    inputs:
-      environment:
-        description: 'Bot Environment'
-        type: choice
-        options:
-          - prd
-          - tst
-          - custom
-          - straykids
-          - nayeon
-        required: true
-      publish: # Publish image to GHCR
-        description: 'Publish image to GHCR?'
-        required: false
-        type: boolean
-        default: true
   push:
-    branches: [testing, production]
-
-permissions:
-  contents: read
-  packages: write
-
-env:
-  # Uses docker.io for Docker Hub if empty
-  REGISTRY: ghcr.io
-  # github.repository as <account>/<repo>
-  IMAGE_NAME: ${{ github.repository }}
-  REPOSITORY_NAME: ${{ github.event.repository.name }}
+    branches:
+      - main
+    paths:
+      - 'src/**'
+      - 'Dockerfile'
+      - '.github/workflows/continuous-delivery.yml'
+      - 'package.json'
 
 jobs:
-  Publish:
```

Comment:
What if we want to deploy the testing environment? This looks like its only made for the production branch?